### ISSUE TITLE
GOV.UK Frontend accessibility improvements

### ISSUE THREAD

**Issue by timpaul** [alphagov/govuk-frontend#1385]
## Must
- [ ] Review if `aria-label` in header navigation is necessary #1340
- [x] Conditional reveals with an input not being announced on JAWS #935
- [x] Conditionally revealing content with checkboxes not announcing collapsed/expanded state on macOS with VoiceOver #989
- [x] Conditional reveal radio component has aXe errors #979
- [x] Fieldset legend text can be clipped when using a custom or fallback font #1239
- [x] Descenders clipped from letters such as qyp when select boxes are in an error state #1534 
- [x] Form elements do not scale well if only font size increased #1519
- [x] Error Summary link handling can result in obscured inputs #1544

## Should
- [ ] All caps text in the tag component is harder to read #1504
- [x] Dragon Accessibility issue with DOB field #1101
- [x] Summary list action links visually hidden text are read out without spaces in screen readers #1643
- [x] Investigate adding 'aria-invalid' attribute on form inputs that have errors #1168
- [x] Hint for fieldset not announced if first input has hint #1696
- [ ] Link colour change on hover not clear #1417
- [x] Screen-readers may not 'announce' lists if 'list-style: none' is used #1471
- [x] Investigate Dynamic Type to improve accessibility for iOS users #882
- [x] Details component not announcing collapsed/expanded state on iOS with VoiceOver #988 
- [x] Investigate why role="group" is applied to details component automatically #1483 
- [x] Character count count message is not associated with textarea #1106
- [x] Character count remaining characters message not visible #1406
- [x] Investigate colour contrast issues with secondary text colour #1134

## Could
- [x] Consider making tabs `<section>` into `<div>` #1339
- [ ] Tabs interface needs better way to get to panel content #860
- [x] Wrapping component text with `<p>` where semantics are missing #1611
- [x] Investigate using more robust method for detecting updates to input fields in Dragon #1028

**Comment by automaticdreams** [comment]
automated testing with axe flags .govuk-footer__copyright-logo as an issue because it cannot calculate contrast as a background image is used without a background colour

**Comment by 36degrees** [comment]
Closing in favour of the accessibility label.
